src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -535,7 +536,7 @@ public sealed class AdoConnector {
                     _logTracer.AddTags(new List<(string, string)> { ("MatchingWorkItemIds", $"{workItem.Id}") });
                     _logTracer.LogInformation("Found matching work item");
                 }
-                if (IsADODuplicateWorkItem(workItem)) {
+                if (IsADODuplicateWorkItem(workItem, duplicateFields)) {
```

Comment:
Continuing from earlier comment, we can pass `_config.AdoDuplicateFields` here and remove the `duplicateFields` argument.

Reply:
whipped up a quick PR with these changes
https://github.com/microsoft/onefuzz/pull/3473